data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
+		word
+			`"`
+news "unfettered hai after wanderers invasion regret"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"I had to shoot down a Wanderer ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
+			"The Wanderers hardly try to fight back. It makes me wonder if they want us to conquer their planets."
+			"I don't think it's right that we attack those that have surrendered. What honor is there in killing civilians?"
+			"I came here to do battle with warriors, not with merchants and cowardly birds."
+			"What challenge is there in killing the weak? I desire to battle with those who are worthy."
+			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and threaten to kill me."
+			"I was held prisoner on a Wanderer world. They fed me and took care of me. The only thing they asked me to do in return was to teach them the Hai langauge. If it were the other way around, I'd be dead."
+			"I was held prisoner on a Wanderer world. They said that we were welcome to take their planets, if it was their fate. It must be a trap."
+			"I was held prisoner on a Wanderer world. Even though I attacked them, they were so kind to me. I don't want to fight them anymore, I'd feel too guilty."
+		word
+			`"`
+
+news "unfettered hai no dialog"
+	location
+		government "Hai (Unfettered)"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			"A Hai looks at you and shouts something in their language. You get the impression that it isn't pleasant."
+			"You pass by a Hai with a false tail. You try not to stare as you walk past."
+			"At a booth, you see a Hai selling stolen ship weaponry for outrageous prices."
+			"At a booth, you see a Hai selling various types of unpleasant-smelling substances. Judging by the effects it has on those who consume them, you guess that they're drugs."
+
+news "unfettered hai no dialog group"
+	location
+		government "Hai (Unfettered)"
+	name
+		word
+			"Group of Unfettered"
+			"Group of Hai"
+			"Group of youth"
+	message
+		word
+			"You see a group of Hai confront another. They shout at each other and bring out weapons, before backing off."
+			"You see a group of Hai confront another. They shout and bring out weapons, before lunging at each other. You walk away before they notice you."
+			"You hear yelling in the Hai language. When you turn around, you see that two Hai are facing off against each other, as onlookers egg them on. You turn away before it gets ugly."
+			"A gang of Hai go around, visibly carrying weapons. Older Hai 'donate' money to them."
```

Comment:
This last line feels very incongruous. The Hai are, in general, quite fit very late into their lives (which are long), and the evidence of the Hai's comments about the Unfettered seems to suggest that they're welcome to come home and return to the fold any time. As such I doubt very much that there's a class of older Hai on Unfettered worlds being extorted for protection money.